### PR TITLE
first branch

### PR DESCRIPTION
**Pull request by szcz9p4n**
Initialization newbranch

### CODE REVIEW

**Review by kaczormp4 — APPROVED** [review]

File: nowybranch.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+pierwsza linia
+
+trzecia linia
+piąta linia
```
Comment (kaczormp4) [inline nowybranch.txt]: please delete it !

Comment (kaczormp4) [inline nowybranch.txt]: please delete it !

Comment (szcz9p4n) [reply nowybranch.txt]: ok. i'll do it

File: nowybranch.txt [hunk 2]
```diff
@@ -1,4 +1,7 @@
...
+
+In sodales convallis leo, vel porta turpis porttitor non. Sed eget nibh pellentesque, posuere ante vitae, dignissim quam. Fusce neque mauris, pulvinar eget luctus varius, facilisis nec ipsum. Maecenas et justo id sapien vehicula eleifend. Integer nec vestibulum mi, eu egestas odio. Aliquam sit amet dictum est. Aenean hendrerit risus consectetur, sollicitudin nisi sed, tempor massa. Vestibulum turpis nunc, interdum ut tellus non, congue tincidunt libero. In in pretium felis, eu efficitur nisi. In eget condimentum urna. Sed finibus bibendum molestie.
```
Comment (kaczormp4) [inline nowybranch.txt]: text is too long

**Review by kaczormp4 — APPROVED** [review]

**Pull request merged**